rider/src/main/kotlin/com/jetbrains/rider/plugins/unity/run/configurations/UnityAttachToEditorProfileState.kt

```diff
@@ -42,11 +45,20 @@ class UnityAttachToEditorProfileState(private val exeDebugProfileState : UnityEx
                     debugProcess.initializeDebuggerTask.debuggerInitializingState.advise(lt) {
                         if (it == DebuggerInitializingState.Initialized) {
                             logger.info("Pass value to backend, which will push Unity to enter play mode.")
+                            var prevState = false
                             lt.bracket(opening = {
                                 // pass value to backend, which will push Unity to enter play mode.
+                                prevState = executionEnvironment.project.solution.frontendBackendModel.playControls.play.valueOrDefault(false)
                                 executionEnvironment.project.solution.frontendBackendModel.playControls.play.set(true)
                             }, terminationAction = {
-                                executionEnvironment.project.solution.frontendBackendModel.playControls.play.set(false)
+                                val project = executionEnvironment.project
```

Comment:
Add a comment why this necessary, i.e. if termination happens before the protocol connection is made, the action is lost